entity-framework/core/modeling/query-types.md

```diff
@@ -1,53 +1,49 @@
 ---
-title: Query Types - EF Core
-author: anpete
+title: Keyless Entity Types - EF Core
+author: AndriySvyryd
+ms.author: ansvyryd
 ms.date: 02/26/2018
 ms.assetid: 9F4450C5-1A3F-4BB6-AC19-9FAC64292AAD
 uid: core/modeling/query-types
 ---
-# Query Types
+# Keyless Entity Types
 > [!NOTE]
-> This feature is new in EF Core 2.1
+> This feature is new in EF Core 2.1. Prior to 3.0 they were known as Query Types
 
-In addition to entity types, an EF Core model can contain _query types_, which can be used to carry out database queries against data that isn't mapped to entity types.
+In addition to normal entity types, an EF Core model can contain _keyless entity types_, which can be used to carry out database queries against data that doesn't contain key values.
 
-## Compare query types to entity types
+## Keyless Entity Types Characteristics
```

Comment:
I think our standard is for lower-case after the first word in non-toplevel headings